sponsors.html

```diff
@@ -4,7 +4,19 @@
 ---
 <div class="container white-text">
   <h1 class="text-accent-4 center">Sponsor</h1>
-
+  <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black update clemson-orange-text">Online Update for Sponsors</div>
+      <div class="collapsible-body">
+        <p>With CUCTF 2020 switching to an online infrastructure, all Silver, Gold, and Diamond sponsors will:</p>
+        <ul class="browser-default">
+          <li>Be provided private online meeting lobbies to connect with students and present information about their organization</li>
+          <li>Maintain sponsorship status into the CUCTF 2021 Event</li>
```

Comment:
Proper noun capitalization nitpick

```suggestion
          <li>Maintain sponsorship status into the CUCTF 2021 event</li>
```